packages/gatsby-remark-prismjs/src/escape-html.js

```diff
@@ -0,0 +1,15 @@
+const isEmpty = require(`lodash/isEmpty`)
+
+module.exports = (code, additionalEscapeCharacters = {}) => {
+  if (isEmpty(additionalEscapeCharacters)) return code
+  const escapedChars = char => additionalEscapeCharacters[char]
+  const chars = Object.keys(additionalEscapeCharacters)
+
+  const charsRe = new RegExp(`[${chars.join()}]`, `g`)
+
+  const rehasUnescapedChars = new RegExp(charsRe.source)
```

Comment:
Could you explain why you create this regex again? I don't quite understand what you are trying to do with this and with the following .test.

Reply:
To be honest, I took this out of lodash's escape function. To me, it appears to be an optimization. I haven't dug into the `.source` property of a `RegExp` object but I'd infer that it returns the inverse regex query when passed into another RegExp obj. Happy to change. 

source: https://github.com/lodash/lodash/blob/master/escape.js#L12

Reply:
also, the test was not to test that line. It was to test the if statement at `ln4`

Reply:
As explained above, the RegExp.source line seems to be a lodash optimization. I'm not one for blindly grabbing code and trusting it but the lodash library has had a lot of eyes on it. I feel safe borrowing from them.

Additionally, due to the changes I've made against the first suggestion I have changed the unit tests.